tests/Unit/Domain/Test_BlockId.cpp

```diff
@@ -0,0 +1,26 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <string>
+
+#include "Domain/BlockId.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+SPECTRE_TEST_CASE("Unit.Domain.BlockId", "[Domain][Unit]") {
+  domain::BlockId id{10};
+  CHECK(id.get_index() == 10);
+  CHECK(id == domain::BlockId{10});
+  CHECK(id != domain::BlockId{7});
+  CHECK(get_output(id) == "[10]");
+  CHECK(id++ == domain::BlockId{11});
```

Comment:
This is the wrong answer.  Should be 10.

Reply:
Derp, done. I also added extra check statements after post increment and decrement to check that the id itself updated.